wpilibc/src/main/native/cpp/ADIS16448_IMU.cpp

```diff
@@ -541,15 +489,15 @@ uint16_t ADIS16448_IMU::ReadRegister(uint8_t reg) {
   m_spi->Write(buf, 2);
   m_spi->Read(false, buf, 2);
 
-  return ToUShort(buf);
+  return (static_cast<uint16_t>(buf[0]) << 8) | buf[1];
 }
 
 /**
  * This function writes an unsigned, 16-bit value into adjacent 8-bit addresses
  *via SPI. The upper and lower bytes that make up the 16-bit value are split
- *into two unsined, 8-bit values and written to the upper and lower addresses of
- *the specified register value. Only the lower (base) address must be specified.
- *This function assumes the controller is set to standard SPI mode.
+ *into two unsigned, 8-bit values and written to the upper and lower addresses
+ *of the specified register value. Only the lower (base) address must be
+ *specified. This function assumes the controller is set to standard SPI mode.
```

Comment:
As long as you're touching this, clean up the comment to have a space after each asterisk.